### PR TITLE
Add a helper class that could contains the commonly used helper method.

### PR DESCRIPTION
**Pull request by jiaxi312**
We could add some helper method or variables that will be used in multiple  servlets to this helper class.
Now, I'm only thinking the Datastore and Gson will be used a lot in multiple servlets, we could add more later on.

### CODE REVIEW

File: easylinks/src/main/java/com/google/step/servlets/ServletHelper.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
```
Comment (xiaoluLucy814) [inline easylinks/src/main/java/com/google/step/servlets/ServletHelper.java:1]: should be `package com.google.step.servlets;`

Comment (jiaxi312) [reply easylinks/src/main/java/com/google/step/servlets/ServletHelper.java]: Fixed！

File: easylinks/src/main/java/com/google/step/servlets/ServletHelper.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.gson.Gson;
+import javax.servlet.http.HttpServletRequest;
+
+/** Class contains the common used helper method for servlets.*/
+public final class ServletHelper {
+
+  /* The static store service that can be used to all servlets */
+  protected final static DatastoreService DEFAULT_DATASTORE_SERVICE
+                            = DatastoreServiceFactory.getDatastoreService();
```
Comment (zixiyan) [inline easylinks/src/main/java/com/google/step/servlets/ServletHelper.java:13]: I think we should not break at ```=```.

Comment (jiaxi312) [reply easylinks/src/main/java/com/google/step/servlets/ServletHelper.java]: Done！

**Comment by xiaoluLucy814** [comment]
Maybe we can add a function to convert an arraylist into json in this helper class?
Something like `protected static final <T> String convertToJson(ArrayList<T> messages)`

**Comment by jiaxi312** [comment]
> Maybe we can add a function to convert an arraylist into json in this helper class?
> Something like `protected static final <T> String convertToJson(ArrayList<T> messages)`

I'll include the helper method, but I'm not sure it is right.

**Pull request merged**